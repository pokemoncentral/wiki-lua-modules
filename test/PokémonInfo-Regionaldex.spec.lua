-- "Test cases" for PokémonInfo-BoxAbil

local r = require("PokémonInfo-Regionaldex").regionaldex

-- Standard Pokémon, with different paddings
print(r({ args = { "64" } }))
print(r({ args = { "0079" } }))
print(r({ args = { "094" } }))
-- Not in an oldDex
print(r({ args = { "359" } }))
