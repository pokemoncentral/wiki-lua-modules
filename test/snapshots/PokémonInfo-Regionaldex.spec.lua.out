<span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Kanto|<span class="black-text">Kanto</span>]]'''</div>#064</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Johto|<span class="black-text">Johto</span>]]'''</div>#090</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Hoenn|<span class="black-text">Hoenn</span>]]'''</div>#<span class="explain tooltips" title="040 in terza generazione">041</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Sinnoh|<span class="black-text">Sinnoh</span>]]'''</div>#021</span><div class="width-xl-100"></div><span><div class="small-font">'''[[Elenco Pokémon secondo i Pokédex di Kalos#Pokédex di Kalos Centrale|<span style="color:#D1D1D1">Kalos</span>]]'''</div>#<span class="explain tooltips" title="Centrale">103</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Alola|<span class="black-text">Alola</span>]]'''</div>#<span class="explain tooltips" title="043 in Sole e Luna">050</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex Armatura|<span class="black-text">Armatura</span>]]'''</div>#032</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Hisui|<span class="black-text">Hisui</span>]]'''</div>#059</span>
<span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Kanto|<span class="black-text">Kanto</span>]]'''</div>#079</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Johto|<span class="black-text">Johto</span>]]'''</div>#080</span><span><div class="small-font">'''[[Elenco Pokémon secondo i Pokédex di Kalos#Pokédex di Kalos Costiera|<span style="color:#013D6C">Kalos</span>]]'''</div>#<span class="explain tooltips" title="Costiera">133</span></span><div class="width-xl-100"></div><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Alola|<span class="black-text">Alola</span>]]'''</div>#<span class="explain tooltips" title="037 in Sole e Luna">044</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex Armatura|<span class="black-text">Armatura</span>]]'''</div>#001</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Paldea|<span class="black-text">Paldea</span>]]'''</div>#324</span>
<span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Kanto|<span class="black-text">Kanto</span>]]'''</div>#094</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Johto|<span class="black-text">Johto</span>]]'''</div>#060</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Sinnoh|<span class="black-text">Sinnoh</span>]]'''</div>#071</span><span><div class="small-font">'''[[Elenco Pokémon secondo i Pokédex di Kalos#Pokédex di Kalos Montana|<span style="color:#981128">Kalos</span>]]'''</div>#<span class="explain tooltips" title="Montana">032</span></span><div class="width-xl-100"></div><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Alola|<span class="black-text">Alola</span>]]'''</div>#<span class="explain tooltips" title="063 in Sole e Luna">075</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Galar|<span class="black-text">Galar</span>]]'''</div>#143</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Hisui|<span class="black-text">Hisui</span>]]'''</div>#138</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Paldea|<span class="black-text">Paldea</span>]]'''</div>#70</span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Nordivia|<span class="black-text">Nordivia</span>]]'''</div>#120</span>
<span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Hoenn|<span class="black-text">Hoenn</span>]]'''</div>#<span class="explain tooltips" title="152 in terza generazione">159</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Sinnoh|<span class="black-text">Sinnoh</span>]]'''</div>#209<sup>[[Pokémon Platino|<span style="color: #999999;">'''Pt'''</span>]]</sup></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Unima|<span class="black-text">Unima</span>]]'''</div>#<span class="explain tooltips" title="Non disponibile in Nero e Bianco">216</span></span><div class="width-xl-100"></div><span><div class="small-font">'''[[Elenco Pokémon secondo i Pokédex di Kalos#Pokédex di Kalos Costiera|<span style="color:#013D6C">Kalos</span>]]'''</div>#<span class="explain tooltips" title="Costiera">009</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex di Alola|<span class="black-text">Alola</span>]]'''</div>#<span class="explain tooltips" title="245 in Sole e Luna">322</span></span><span><div class="small-font">'''[[Elenco Pokémon secondo il Pokédex Corona|<span class="black-text">Corona</span>]]'''</div>#107</span>
